gillespy2/solvers/numpy/tau_hybrid_solver.py

```diff
@@ -827,6 +827,11 @@ def run(self=None, model=None, t=None, number_of_trajectories=1, increment=None,
             "interval" specifies seconds between displaying.
             "clear_output" specifies if display should be refreshed with each display
         :type live_output_options:  str
+
+        :param timeout: If set, if simulation takes longer than timeout, will exit.
+        :type timeout: int
+        
+        :returns: a list of each trajectory simulated.
```

Comment:
solver run functions always return result objects not lists. Also missing the rtype tag for the return type